client/src/components/page/Sidebar/index.tsx

```diff
@@ -68,7 +73,14 @@ export const SidebarComponent: React.FC<SidebarProps> = (props: SidebarProps) =>
               </Link>
             </Menu.Item>
           )}
-          <Menu.SubMenu key="admin-locations" title="Locations">
+          {ENABLE_PLANS && (
+            <Menu.Item key="plans">
+              <Link to={PLANS_LIST_VIEW_URL} className="admin-link">
```

Comment:
@kahummer whatof the subMenus that are supposed to go here, are those going to be addressed in this PR?

Reply:
@p-netm I think those can go into another pr once we have create and edit components merged in